website/translated_docs/ar/kubernetes.md

```diff
@@ -86,7 +86,7 @@ kubectl create configmap verdaccio-config --from-file ./config.yaml
 Now you can deploy the Verdaccio Helm chart and specify which configuration to use:
 
 ```bash
```

Comment:
Thanks for the big effort, but actually only `docs/kubernetes.md` is need it to be changed :) details are on the CONTRIBUTING guidelines, other files are automated.

Reply:
Thanks, that's a relief, wish I'd known before, then I'd probably made a pr without being asked.

Force-pushed without the translations.